en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/02-creating-pages/01-creating-pages-intro.markdown

```diff
@@ -41,7 +41,9 @@ From *Pages* you can do several things:
 ![Figure 2: Understanding the options on Site Pages.](../../../../../images/site-pages-breakdown.png)
 
 Adding a child page creates child pages in the hierarchy below the page you've 
-selected. You can nest pages as deep as you like. 
+selected. You can nest pages as deep as you like. The friendly URLs generated
+for a new page are not hierarchical. This means that a top-level page and a
```

Comment:
I'm not sure this explanation is clear enough.

First, I think we might want to add "by default" in "for a new page are not hierarchical by default". Or otherwise explain that Liferay does not force a hierarchy in the URL, but you are free to follow a hierarchical structure if desired.

Also, I find it hard to understand what the final sentence means. I believe this would be much more clear with some examples.

Finally, knowing that this behavior of the product is causing confusion because many other CMSs behave differently, I believe we should highlight this message more. One way could be by making it a Note. What do you think?

Reply:
Hey @JorgeFerrer; sure I can make this more prominent and provide examples. I'll draft up a more in-depth explanation and send to you for review today.

Reply:
Cool, thanks